src/main/osd/osd.c

```diff
@@ -492,8 +493,9 @@ static void osdUpdateStats(void)
     }
 #endif
 
-    value = getBatteryVoltage();
-    if (stats.min_voltage > value) {
+    value = osdConfig()->stat_show_cell_value ? getBatteryAverageCellVoltage() : getBatteryVoltage();
+    if (stats.min_voltage > value)
+    {
```

Comment:
I would place the { after the `if (comparison)`

Reply:
Correct. Betaflight uses 1TBS for braces. Please amend.